components/FriendsList/index.js

```diff
@@ -1,7 +1,12 @@
+import Link from "next/link";
+
 export default function FriendsList({ friendName }) {
   return (
     <section>
-      <div>{friendName}</div>
+      <hr />
+      <div>

```

Comment:
you can get rid off the div element and the Link Element can also be a self closed Tag <Link/>

Reply:
I need the div element because i want to add buttons next to the friends-names, which i can use for deleting or show different information. with a div, this information will be shown in one line with the friend name.